docs/content/preview/explore/fault-tolerance/_index.md

```diff
@@ -65,12 +92,6 @@ For more information, see the following:
 
 {{<index/block>}}
 
-  {{<index/item
-    title="High availability during node and zone failures"
-    body="Continuously serve requests in the event of unplanned outages."
-    href="macos/"
-    icon="/images/section_icons/explore/zero_downtime.png">}}
-
   {{<index/item
```

Comment:
With all the rest being note blocks, should probably convert this one as well and put it after line 84